ports/esp32s2/mpconfigport.mk

```diff
@@ -21,7 +21,7 @@ CIRCUITPY_COUNTIO = 0
 CIRCUITPY_FREQUENCYIO = 0
 CIRCUITPY_I2CPERIPHERAL = 0
 CIRCUITPY_ROTARYIO = 0
-CIRCUITPY_RTC = 0
+CIRCUITPY_RTC = 1
```

Comment:
Could you try removing this line instead? I think it will be on by default.

Reply:
That did work! I retested on the Kaluga.

Reply:
Don't change FULL_BUILD. Delete this line.

Reply:
OK.. moving slower. I reverted that change. 

With CIRCUITPY_RTC = 0 no RTC is available in the image so I think the original PR was the correct change. If I'm not understanding let me know.

Reply:
@tannewt  Yes, when I remove the CIRCUITPY_RTC line completely RTC is present in the build.